core/templates/dev/head/pages/exploration-editor-page/statistics-tab/services/learner-action-render.service.ts

```diff
@@ -68,25 +75,21 @@ oppia.factory('LearnerActionRenderService', [
      * @param {int} timeSpentInStateSecs.
      * @param {string} currentStateName.
      * @param {int} actionIndex.
+     * @param {Interaction} interaction.
      * @returns {string}
      */
     var renderAnswerSubmitActionHTML = function(
         answer, destStateName, timeSpentInStateSecs, currentStateName,
-        actionIndex) {
-      var htmlString;
-      if (currentStateName === destStateName) {
-        htmlString =
-          '<span class="oppia-issues-learner-action">' + actionIndex +
-          '. Submitted answer "' + answer + '" in card "' + currentStateName +
-          '".</span>';
-      } else {
-        htmlString =
-          '<span class="oppia-issues-learner-action">' + actionIndex +
-          '. Submitted answer "' + answer + '" and moved to card "' +
-          destStateName + '" after spending ' + timeSpentInStateSecs +
-          ' seconds on card "' + currentStateName + '".</span>';
-      }
-      return htmlString;
+        actionIndex, interaction) {
+      var el = $('<answer-submit-action>');
+      el.attr('answer', HtmlEscaperService.objToEscapedJson(answer));
+      el.attr('destStateName', destStateName);
+      el.attr('timeSpentInStateSecs', timeSpentInStateSecs);
+      el.attr('currentStateName', currentStateName);
+      el.attr('actionIndex', actionIndex);
+      el.attr('interactionId', interaction.id);
+      el.attr('interactionCustomizationArgs', interaction.customizationArgs);
```

Comment:
I'm slightly confused about this. How does interaction.customizationArgs get converted to a string? Can you add unit tests that include customization args?

Reply:
Thanks for catching this. The customisation args are needed only for MultiplechoiceInput, ItemSelectionInput where there are choices. The unit test exposed that bug and I fixed it. Done now.